docroot/themes/humsci/humsci_basic/humsci_basic.theme

```diff
@@ -129,6 +129,15 @@ function humsci_basic_preprocess_page(&$vars) {
   $config = \Drupal::config('system.site');
   $vars['site_name'] = $config->get('name');
   $vars['site_slogan'] = $config->get('slogan');
+  
+  if ($route === 'entity.node.canonical') {
+    $node = $vars['node'];
+    // Determine if Layout Builder is enabled for current node. Based on this,
+    // the default main content id will be added/removed.
+    $vars['lb_enabled'] = \Drupal::service('entity_display.repository')
+      ->getViewDisplay('node', $node->bundle())
+      ->isLayoutBuilderEnabled();
```

Comment:
This is pretty heavy. You can just check if `$node->get('layout_builder__layout')->count()` to check for customized layout builder settings. Otherwise it should be expected every content type has layout builder on the default display.

Reply:
Fixed. Removed the check altogether because I was checking for layout builder enabled for the display, not overrides. Added a check on the templates to check for node pages.